subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/AdbDevice.kt

```diff
@@ -41,14 +41,17 @@ data class AdbDevice(
             retriesCount = 5,
             delaySeconds = 3,
             block = { attempt ->
-                log("Attempt $attempt: getting ro.build.version.sdk")
+                logger.debug("Attempt $attempt: getting ro.build.version.sdk")
                 loadProperty(
                     key = "ro.build.version.sdk",
                     cast = { it.toInt() }
                 )
             },
-            attemptFailedHandler = { attempt, throwable ->
-                log("Attempt $attempt: reading ro.build.version.sdk failed. Reason: ${throwable.message}")
+            attemptFailedHandler = { attempt, _ ->
+                logger.info("Attempt $attempt: reading ro.build.version.sdk failed")
+            },
+            actionFailedHandler = { throwable ->
+                logger.warn("Failed reading ro.build.version.sdk", throwable)
             }
```

Comment:
I've added a final handler to log the full exception.
Found too verbose to log all details on every attempt.